build-system/common/check-package-manager.js

```diff
@@ -349,17 +366,22 @@ function checkPythonVersion() {
   }
 }
 
+/**
+ * Runs checks for the package manager and tooling being used.
+ * @return {Promise}
+ */
 async function main() {
-  // Yarn is already used by default on Travis, so there is nothing more to do.
-  if (process.env.TRAVIS) {
-    return 0;
+  // NPM is already used by default on Travis and Github Actions, so there is
+  // nothing more to do.
+  if (process.env.TRAVIS || process.env.GITHUB_ACTIONS) {
+    return Promise.resolve();
```

Comment:
Since it's an async function, `return;` is fine here (the result will be a Promise.resolve(undefined) anyway)